src/tss2-fapi/ifapi_eventlog.c

```diff
@@ -313,7 +313,6 @@ ifapi_eventlog_append_finish(
 
         r = ifapi_json_IFAPI_EVENT_serialize(&eventlog->event, &event);
         if (r) {
-            json_object_put(eventlog->log);
```

Comment:
Then we should we remove line 331 also.

Reply:
I will add:  eventlog->log = NULL;
because in the success case the json_object_put would not be called if the line is removed.
Thanks for the hint.